### ISSUE TITLE
Story: The SwiftUI View for representing the HTML is missing.

### ISSUE THREAD

**Issue by maetzemax** [maetzemax/HTML-to-Swift#1]
tl:dr; The user is expecting to get an SwiftUI View based on HTML String. 

**User description**
The user would like to add a SwiftUI View, which can represent HTML body-text as SwiftU componentI. To achieve that he'd like to import the package and add the View to the body, by only parsing an HTML string.

**Technical description**
Create a algorithm which can extract the corresponding informations of the HTML string.
There can be the problem that for example a bold formatting is inside an italic formatting. Be aware of that and iterate properly.
Use the informations to convert them to SwiftUI components. Those components should be placed together to present the whole view.
The Challenge here is to get the right order done.
